prepare_data/src/test/groovy/org/orbisgis/bdtopo/BDTopoGISLayersTest.groovy

```diff
@@ -49,7 +49,7 @@ class BDTopoGISLayersTest {
     @Test
     @EnabledIfSystemProperty(named = "test.bdtopo", matches = "true")
     void initTypes(){
-        H2GIS h2GISDatabase = H2GIS.open(bdTopoDb-".mv.db", "sa", "")
+        H2GIS h2GISDatabase = H2GIS.open(bdTopoDb.absolutePath-".mv.db", "sa", "")
```

Comment:
if there is no identification on the database, you can use `H2GIS.open(String path)`

Reply:
I am not sure I got it

Reply:
If the database haven't been created with custom username and password (other than `"sa"` and `""`), you can use the method `H2GIS.open(String path)` which uses `"sa"` and `""` as default credentials

Reply:
This deletion does not work. I got the following error message:

> ERROR org.orbisgis.datamanager.h2gis.H2GIS - Unable to create the DataSource.
Mauvais nom dutilisateur ou mot de passe
Wrong user name or password [28000-197]

Reply:
Ok, so keep it as it is.